adlfs/spec.py

```diff
@@ -82,7 +82,9 @@ def wrapper(response):
 
         current = response.context.get(key)
         total = response.context["data_stream_total"]
-        if not sent_total:
+        if current is None:
+            return
+        if not sent_total and total is not None:
```

Comment:
`sent_total` is not even being used here.

Should probably be set inside to `True`, but 🤷🏽.